pkg/daemon/struct.go

```diff
@@ -34,6 +34,9 @@ var (
 	SyncInterval              = "katalog-sync.wish.com/sync-interval"     // How frequently we want to sync this service
 	ConsulServiceCheckTTL     = "katalog-sync.wish.com/service-check-ttl" // TTL for the service checks we put in consul
 	ContainerExclusion        = "katalog-sync.wish.com/container-exclude" // comma-separated list of containers to exclude from ready check
+
+	// Meta keys that are used as per-service flags
```

Comment:
Should make this clear this is a metadata on the consul service (since all the things above are all k8s annotations; its a bit unclear).

Reply:
done